src/main/drivers/accgyro_fake.c

```diff
@@ -28,26 +50,55 @@
 #include "accgyro.h"
 #include "accgyro_fake.h"
 
-
 static int16_t fakeGyroADC[XYZ_AXIS_COUNT];
 
+#if defined(SIMULATOR_BUILD) && defined(SIMULATOR_MULTITHREAD)
```

Comment:
Is it really necessary to make this dependent on `SIMULATOR_BUILD`? Testing only `SIMULATOR_MULTITHREAD` may be enough. And in future, mutexes may be used even without simulator